produce/lib/produce/merchant.rb

```diff
@@ -72,12 +58,50 @@ def app_identifier
       Produce.config[:app_identifier].to_s
     end
 
+    def pluralize(singular, arr)
+      return singular if arr.count == 1
+
+      "#{singular}s"
+    end
+
     def merchant_exists?(identifier)
-      Spaceship.merchant.find(identifier, mac: Produce.config[:platform].to_s == 'mac') != nil
+      find_merchant(identifier)
+    end
+
+    def detect_merchant_identifier
+      self.class.detect_merchant_identifier
+    end
+
+    def self.detect_merchant_identifier(config = Produce.config)
+      identifier = config[:merchant_identifier] || input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ", ":merchant_identifier option is required")
+      prepare_identifier(identifier)
+    end
```

Comment:
Sorry if I am missing something here - is there a reason why we have this as a class level method and an instance method?